packages/wallet/src/redux/protocols/reducer-helpers.ts

```diff
@@ -11,7 +11,9 @@ import * as magmoWalletClient from 'magmo-wallet-client';
 import { getLastCommitment, nextParticipant, Commitments } from '../channel-store';
 import { Commitment } from '../../domain';
 import { sendCommitmentsReceived, ProtocolLocator } from '../../communication';
+import * as comms from '../../communication';
 import { ourTurn as ourTurnOnChannel } from '../channel-store';
+import _ from 'lodash';
```

Comment:
Is this introducing `lodash` as a dependency? I only recall seeing it in test files.

Reply:
We're making use of `lodash` in a couple of (non-test) places. Is there an issue having `lodash` as a dependency?